bindings/jni/src/main/java/org/zeromq/czmq/Zsys.java

```diff
@@ -367,31 +367,29 @@ public int ipv6 () {
         return __ipv6 ();
     }
     /*
-    Set network interface name to use for broadcasts, particularly zbeacon.    
+    Set network interface name to use for broadcasts, particularly zbeacon.
     This lets the interface be configured for test environments where required.
-    For example, on Mac OS X, zbeacon cannot bind to 255.255.255.255 which is  
-    the default when there is no specified interface. If the environment       
-    variable ZSYS_INTERFACE is set, use that as the default interface name.    
-    Setting the interface to "*" means "use all available interfaces".         
+    For example, on Mac OS X, zbeacon cannot bind to 255.255.255.255 which is
+    the default when there is no specified interface. If the environment
+    variable ZSYS_INTERFACE is set, use that as the default interface name.
+    Setting the interface to "*" means "use all available interfaces".
     */
     native static void __setInterface (String value);
     public void setInterface (String value) {
         __setInterface (value);
     }
     /*
     Return network interface to use for broadcasts, or "" if none was set.
-    Note that "interface" is among reserved Java keywords, so it had to be
-    renamed here manually to "getInterface".
     */
     native static String __interface ();
-    public String getInterface () {
```

Comment:
hrm. i'm literally just re-running [or trying to] gsl project.xml w/o any "changes" (from me). have to see where that's coming from... trying to get things in-sync before i PR my actual changes (to avoid this kinda thing :)).